Shared/sdk/SharedUtil.Misc.hpp

```diff
@@ -2063,6 +2065,18 @@ namespace SharedUtil
         return false;
     }
 
+    std::vector<std::string> Explode(std::string const& s, char delim)
```

Comment:
Is "Split" common name for this opeartion?

Reply:
"Explode" would be the correct equivalent in PHP (sorry, I use PHP daily). It's called split in JavaScript, so either is fine imo.